<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screen_DateTime</class>
 <widget class="QWidget" name="Screen_DateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_days" native="true">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>190</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>639</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 235, 255);</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_month">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ENERO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelAnimated" name="l_am_pm">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(227, 227, 227);
</string>
   </property>
   <property name="text">
    <string>am</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 235, 255);</string>
   </property>
   <property name="text">
    <string>M</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 235, 255);</string>
   </property>
   <property name="text">
    <string>M</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 235, 255);</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_year">
   <property name="geometry">
    <rect>
     <x>533</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>2020</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 235, 255);</string>
   </property>
   <property name="text">
    <string>L</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_minutes_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(227, 227, 227);
</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>579</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 235, 255);</string>
   </property>
   <property name="text">
    <string>J</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>609</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(153, 235, 255);</string>
   </property>
   <property name="text">
    <string>V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelStateAnimated" name="pb_ok">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(227, 227, 227);
     border: 2px solid #FFFFFF;
     border-radius: 5px;

 }</string>
   </property>
   <property name="text">
    <string>ACEPTAR</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="pb_ok_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>74</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Bold</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 5px;
	background-color: rgb(153, 235, 255);
	color: rgb(36, 41, 46);
 }</string>
   </property>
   <property name="text">
    <string>FECHA / HORA</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_minutes_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>3</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(120, 120, 120);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="pb_ok_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Medium</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/clock.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_down_year">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>75</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_izq.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_down_hour">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>245</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_abajo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_down_minutes">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>245</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_abajo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_up_year">
   <property name="geometry">
    <rect>
     <x>588</x>
     <y>75</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_der.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_up_month">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>115</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>51</width>
     <height>53</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_der.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_up_hour">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_arriba.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_up_minutes">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_arriba.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyLabelLongPressAnimated" name="l_down_month">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>115</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>51</width>
     <height>53</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/flecha_izq.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>802</width>
     <height>482</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(17, 21, 25);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_minutes">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>212</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(17, 21, 25);
border-radius: 5px;
color: rgb(153, 235, 255);</string>
   </property>
   <property name="text">
    <string>05</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_hour">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>212</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(17, 21, 25);
border-radius: 5px;
color: rgb(153, 235, 255);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_version">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>0</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/version.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/version.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>widget_days</zorder>
  <zorder>label_14</zorder>
  <zorder>l_month</zorder>
  <zorder>l_am_pm</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_8</zorder>
  <zorder>l_year</zorder>
  <zorder>label_9</zorder>
  <zorder>l_minutes_2</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>pb_ok</zorder>
  <zorder>pb_ok_2</zorder>
  <zorder>l_minutes_3</zorder>
  <zorder>pb_ok_3</zorder>
  <zorder>l_down_year</zorder>
  <zorder>l_down_hour</zorder>
  <zorder>l_down_minutes</zorder>
  <zorder>l_up_year</zorder>
  <zorder>l_up_month</zorder>
  <zorder>l_up_hour</zorder>
  <zorder>l_up_minutes</zorder>
  <zorder>l_down_month</zorder>
  <zorder>l_minutes</zorder>
  <zorder>l_hour</zorder>
  <zorder>l_version</zorder>
  <zorder>label_5</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabelStateAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelstateanimated.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabelAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelanimated.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabelLongPressAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabellongpressanimated.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
